Tools/MiniBrowser/haiku/BrowserWindow.cpp

```diff
@@ -172,7 +172,7 @@ void BrowserWindow::MessageReceived(BMessage* message)
 			break;
 
 		case DID_CHANGE_PROGRESS:
-			LoadingProgress(fWebView->didChangeProgress());
+			LoadingProgress(fWebView->progress());
```

Comment:
ideally, the DID_CHANGE_PROGRESS message would contain the current progress, saving us one call back into WebKit here

Reply:
Well, looks like there would be one call into WebKit no matter what, so, no efficiency gains afaik. Oh well, I think it's more intuitive this way anyway.